recipes/default.rb

```diff
@@ -29,7 +32,7 @@
 
 directory node.loggly.tls.cert_path do
   owner 'root'
-  group 'syslog'
+  group node.rsyslog.priv_group || 'root'
```

Comment:
I should have read your commit first. I understand better now. That looks better having it in the attributes.rb. :clap: